### PR TITLE
Stripe gem installed

### PR DESCRIPTION
**Pull request by wJoenn**
Clicking on an event in the scroll will show a booking card
image

Clicking on book now will redirect to Stripe
image

Then either we cancel or we pay, in both case we are redirected to the home page with a flash
image

### CODE REVIEW

**Review by haqcked — COMMENTED** [review]
Where can i find the code for this?
image

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -18,9 +21,45 @@ def cancel
...
+    )
+  end
+
+  def stripe_customer
+    Stripe::Customer.create(
+      name: current_user.full_name,
+      email: current_user.email,
+      description: "Customer id: #{current_user.id}"
+    )
+  end
```
[the remark this answers is not in the document]
Comment (wJoenn) [reply app/controllers/bookings_controller.rb:64]: Sorry Haq I missed those comments.
The screenshot you share is a page from stripe, our website redirectes to stripe for payment then stripê redirect to our website after payment.
Line 13 is the one redirecting.

Stripe::Checkout::Session.create() and Stripe::Customer.create() are part of stripe's gem indeed so I did not made those.
The infos on how to use them are available on the documentation yes

[the remark this answers is not in the document]
Comment (haqcked) [reply app/controllers/bookings_controller.rb:64]: I wonder if we can choose the theme of the interface of the page from stripe. Just thinking, imagine if we have a light-themed web app then suddenly when paying the dark-themed page comes up from stripe, wouldn't it be against the UX of our web app?

**Comment by haqcked** [comment]
btw this branch has conflicts

File: app/controllers/bookings_controller.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (wJoenn) [reply app/controllers/bookings_controller.rb:64]: The page is actually white 😄 I'm the one using a dark theme
image

The red part can be edited with tags inside the CookingsController#stripe_checkout_session so we could be adding a description, a picture etc but I haven't done that yet 🤔

**Review by k-kenney — APPROVED** [review]
It looks to me like everything is in order.

**Pull request merged**